### ISSUE TITLE
Should I use AFSreconstruction.R from SurfaceReconstruction or cgalMeshes? 

### ISSUE THREAD

**Issue by Jsalas424** [stla/cgalMeshes#4]
Hello,

Thank you for developing this package, I'm currently doing an academic research project and needed to deploy CGAL's AFS reconstruction and have had beautiful results with your implementation in the SurfaceReconstruction repo.

I see you have slightly different versions of AFSreconstruction.R in both the [cgalMeshes](https://github.com/stla/cgalMeshes/blob/main/R/AFSreconstruction.R) and [SurfaceReconstruction](https://github.com/stla/SurfaceReconstruction/blob/master/R/AFSreconstruction.R) repo's so I wanted to ask which one you recommend I use? I want to make sure I'm using the most correct/up-to-date deployment prior to publication and citing your work.

**Comment by Jsalas424** [comment]
Additionally, I'm interested in the mesh [smoothing](https://github.com/stla/MeshesOperations/blob/master/R/smoothing.R) function you have in the [MeshesOperations](https://github.com/stla/MeshesOperations) repo. Is there any intent to include those code in cgalMeshes, or will it exist as a standalone?

**Comment by stla** [comment]
Hello,

Basically the AFS reconstruction is the same in the two packages. But **SurfaceReconstruction** is not on CRAN and will not be maintained.

The smoothing throws a C++ warning not acceptable by CRAN, so I can't include it in **cgalMeshes** (unless I do a Github-only version). I think there's the same smoothing operation in the R package **Rvcg**.

**Comment by Jsalas424** [comment]
Thank you!

**Comment by stla** [comment]
Hello @Jsalas424 

I decided to develop the package in a non-master branch (the **github** branch) to add the features unacceptable for CRAN. I already added the Poisson reconstruction, the shape smoothing, and the Minkowski sum. To install the package from this branch:

```r
remotes::install_github("stla/cgalMeshes@github")
```

This branch will be maintained.

Also take a look at the new features (available in both branches): [NEWS.md](https://github.com/stla/cgalMeshes/blob/main/NEWS.md).

Cheers.
